src/data-server/sbvr-server.js

```diff
@@ -138,10 +138,12 @@ export async function setup(app, sbvrUtils, db) {
 						throw new Error('No SE data model found');
 					}
 					const instance = result[0];
+					// prettier-ignore
```

Comment:
This is necessary otherwise prettier will remove the parenthesis and this inlining types into JS does not like that